src/Build/Evaluation/ProjectRootElementCache.cs

```diff
@@ -210,6 +210,7 @@ private bool IsInvalidEntry(string projectFile, ProjectRootElement projectRootEl
         internal override ProjectRootElement Get(string projectFile, OpenProjectRootElement loadProjectRootElement, bool isExplicitlyLoaded,
             bool? preserveFormatting)
         {
+#pragma warning disable format // the release configuration fails as try-catch block is conditioned only for Debug; making the formatting in correct in Release.
```

Comment:
Can we instead move the `#endif` (to 222) and `#if` (to 310) to preserve the block (just without the `try`)?
That way the indetion would be correct in both configuration and we could preserve the styling check.

Reply:
let me try that

Reply:
Fixed